send_status_email

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import subprocess
+import sys
+from textwrap import dedent
+
+from ocflib.misc.mail import send_mail
+
+# TSV or space separated file of username \t # of unexcused absences
+# e.g. abizer   2
+DATA_FILE = sys.argv[1]
```

Comment:
sadly right now it has to be manually created, this is because we have to look at magic words + times in the form and cross reference with the excuses sheet. there's a way to do this smarter but it'd take more time than just going through ~70 students.

in the future, this problem would be solved by having one facilitator have an attendance sign-in sheet and then input attendance records into google sheets or a table somewhere. no magic words/timestamp checking needed, and probably also much easier to cross-reference any excuses

Reply:
Fair enough, I imagined it was a kind of manual process given our setup so far, I just wasn't sure how this was generated exactly, so thanks for the description on that.